### ISSUE TITLE
Crate on crates.io seems to be out-dated

### ISSUE THREAD

**Issue by papricasix** [ogham/rust-ansi-term#28]
Hi, thanks for this very nice crate!

I've noticed, the version on crates.io is out-dated. It is missing
`ansi_term::enable_ansi_support();`

Thanks

**Comment by smt923** [comment]
I needed this function too, so I've resorted to this in my Cargo.toml in the meantime, in case anyone doesn't know Cargo can do this:

```
ansi_term = { git = "https://github.com/ogham/rust-ansi-term", rev = "dee04cbc9745f33be155a23d09e43d0508d5cc52" }```

Obviously not as nice as using the crate but if you need the windows function this should pin it to the (at the time of posting) most recent git commit

**Comment by ogham** [comment]
v0.10.0 is now up! I think I just forgot to publish it last time, sorry about that.

**Comment by smt923** [comment]
I don't know if this is an issue with crates.io at this point but I still only see 0.9.0 published for some reason

**Comment by ogham** [comment]
That's weird, I tried uploading it again and got a 500 error, but it seems to be there. Can you see it published now?

**Comment by ogham** [comment]
Looks like this issue is related: https://github.com/rust-lang/crates.io/issues/1046

I like your vampire avatar btw

**Comment by smt923** [comment]
Yeah I still don't see it: https://i.imgur.com/3kwQVC5.png

And thanks! I noticed that too, it's made it difficult to replace it with anything because it's perfect as it is

**Comment by ogham** [comment]
I published v0.10.1 without any errors in 872ff01a009d8f63df1ea267583e2a6b7beffd88.
